internal/app/pactproxy/proxy.go

```diff
@@ -179,7 +200,9 @@ func (a *api) interactionsWaitHandler(res http.ResponseWriter, req *http.Request
 		}
 
 		log.Infof("waiting for %s", waitFor)
+		log.Infof("CONFIG: delay: %s, duration: %s", a.delay, a.duration)
```

Comment:
wouldn't it be better to use `WithFields` to produce structured logging?

Reply:
Looks like we've accidentally committed some debug code here, we can remove this log line

Reply:
removed